lib/rubocop/cop/rspec/capybara/current_path_expectation.rb

```diff
@@ -14,13 +14,19 @@ module Capybara
         # which ensures that preceding actions (like `click_link`) have
         # completed.
         #
+        # This cop does not support automatic correction in some cases.
+        #
         # @example
         #   # bad
         #   expect(current_path).to eq('/callback')
-        #   expect(page.current_path).to match(/widgets/)
         #
         #   # good
         #   expect(page).to have_current_path("/callback")
+        #
+        #   # bad (not supported autocorrction)
+        #   expect(page.current_path).to match(/widgets/)
```

Comment:
This is being corrected. The one that is not, according to the specs, if where the match is against a variable

Reply:
I guess I was mistaken. Thank you very much. I have changed the case to use variables.